pkg/operatorrules/registry_test.go

```diff
@@ -78,8 +78,9 @@ var _ = Describe("OperatorRules", func() {
 			Expect(err).To(BeNil())
 
 			registeredRules := ListRecordingRules()
-			Expect(registeredRules).To(HaveLen(1))
-			Expect(registeredRules[0].Expr.String()).To(Equal("sum(rate(http_requests_total[10m]))"))
+			Expect(registeredRules).To(HaveLen(2))
+			Expect(registeredRules[0].Expr.String()).To(Equal("sum(rate(http_requests_total[5m]))"))
+			Expect(registeredRules[1].Expr.String()).To(Equal("sum(rate(http_requests_total[10m]))"))
```

Comment:
CI is failing, since this is sorted, lexically `10m` comes first than `5m`, so the order is reversed
you can run the tests locally with `make sanity`, would be easier to catch earlier

Reply:
thanks didn't know the option to make all